background/transport.js

```diff
@@ -261,6 +261,11 @@ export default class Transport {
 
   fetchAccountsForCache (origin) {
     return this.transport.execute('parity_getDappsAddresses', origin)
+      .then(null, () => {
```

Comment:
`Promise` behaves like a monoid, every method (`then`, `catch` and `finally` in the future) returns a new Promise that resolves or rejects just like `new Promise(() => …)` does.

From my memory, not sure: `catch` catches every previous rejection, whereas `b` in `.then(a, b)` does not handle rejections of `a`.

In this specific case, using `catch(f)` should be more idiomatic than `then(null, f)`.